go/teams/loader.go

```diff
@@ -434,6 +435,7 @@ func (l *TeamLoader) load2InnerLockedRetry(ctx context.Context, arg load2ArgT) (
 	ctx, tbs := l.G().CTimeBuckets(ctx)
 	tracer := l.G().CTimeTracer(ctx, "TeamLoader.load2ILR", teamEnv.Profile)
 	defer tracer.Finish()
+	mctx := libkb.NewMetaContext(ctx, l.G())
```

Comment:
Things get tricky when both `ctx` and `mctx` are in scope. See L616. No bug yet. How about going all the way and making this function take `mctx` so that no `ctx` var is in scope?

Reply:
omg that would take a lot of work

Reply:
`s/ctx/mctx.Ctx()` and passthrough methods for `CTimeBuckets` and `CTimeTracer`

Reply:
but sure another thing could be to not bind `mctx` and just do `ret = l.storage.Get(libkb.NewMetaContext(...)...)`

Reply:
took the easy way out